README.md

```diff
@@ -1,6 +1,16 @@
 # DarkSite
 
-## What is DarkSite?
+[![forthebadge](https://forthebadge.com/images/badges/built-with-love.svg)](https://forthebadge.com)
+[![forthebadge](https://forthebadge.com/images/badges/built-by-developers.svg)](https://forthebadge.com)
+[![forthebadge](https://forthebadge.com/images/badges/made-with-javascript.svg)](https://forthebadge.com)
+[![forthebadge](https://forthebadge.com/images/badges/built-with-swag.svg)](https://forthebadge.com)
+
+[![GitHub issues](https://img.shields.io/github/issues/sudo-rgorai/DarkSite?color=Green&style=for-the-badge)](https://github.com/sudo-rgorai/DarkSite/issues)
+![GitHub closed pull requests](https://img.shields.io/github/issues-pr-closed-raw/sudo-rgorai/DarkSite?style=for-the-badge)
+
+
+
+## About Project
 
 It is a chrome extension that customizes your new tab to a minimal dark theme.
```

Comment:
If we are replacing `What is DarkSite` with `About Project` then we should replace `It is a chrome extension...` with `DarkSite is a chrome extension...`